src/core/constants.js

```diff
@@ -75,7 +79,7 @@ export const validAddonTypes = [
 // * '/themes/' -> 'persona'
 export const API_ADDON_TYPES_MAPPING = {
   extensions: ADDON_TYPE_EXTENSION,
-  themes: ADDON_TYPE_THEME,
+  themes: ADDON_TYPE_THEME || ADDON_TYPE_STATIC_THEME,
```

Comment:
Does that work? What's the idea here?

Reply:
At this point I'm not sure this need to be updated (yet) for the updates in the PR. I'll revert this and circle back on this...